user-guide/Cloud_Platform/Catalog/Tutorials/Tutorial_Register_Catalog_Item.md

```diff
@@ -0,0 +1,166 @@
+---
+uid: Tutorial_Register_Catalog_Item
+---
+
+# Registering a new catalog item
+
+This tutorial demonstrates how to add a new catalog item using the [Catalog API](xref:Catalog_Registration).  
+We will be registering our own version of the following example [connector](https://github.com/SkylineCommunications/SLC-C-Example_Rates-Custom).  
+
+For ease of use we will be using [Postman](https://www.postman.com/) to execute the register API calls. Feel free to integrate them afterwards in your workflow, e.g by using them in a github action.
+
+## Prerequisites
+
+- Organization key with permission "Register catalog items".
```

Comment:
Think this prerequisite can be removed since we describe how to get one in step 2

Reply:
i would keep it in because it might take some time for a normal user to obtain a key , its clear up front then he will need a key and doesnt find out about it while started already

Reply:
true, then it can be left as is from my end